### PR TITLE
Installing using chocolatey for windows

### LINKED ISSUE #404: Chocolatey package is not documented
Forking from https://github.com/ManimCommunity/manim/issues/13#issuecomment-685683150

### PR DESCRIPTION
**Pull request by naveen521kk**
## List of Changes
- Fixes #404 
- Create documentation for users to install manim using chocolatey

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

### CODE REVIEW

File: docs/source/installation/win.rst [hunk 1]
```diff
@@ -5,6 +5,18 @@ First, you need to install Chocolatey, which is a package manager for Windows
 systems.  Please refer to `this link <https://chocolatey.org/install>`_ for
 instructions.
 
+Installing using Chocolatey
+***************************
+
+You can install chocolatey very easily using chocolatey. Typing the command install Manim.
```
Comment (leotrs) [inline docs/source/installation/win.rst:11]:
```suggestion
You can install manim very easily using chocolatey, by typing the following command.
```

**Review by leotrs — APPROVED** [review]

File: docs/source/installation/win.rst (hunk 1 above)
Comment (behackl) [reply docs/source/installation/win.rst:11]: This suggestion was not applied?

Comment (naveen521kk) [reply docs/source/installation/win.rst:11]: I think it should be applied. I was drowsy when I was typing that and so it was wrong.

Comment (leotrs) [reply docs/source/installation/win.rst:11]: Lol I left it and then merged this. :joy:  Yes we should make this change.

Comment (naveen521kk) [reply docs/source/installation/win.rst:11]: Who _is _ making an PR for this?

**Pull request merged**